src/gui/properties/trackerlistwidget.cpp

```diff
@@ -367,13 +367,13 @@ void TrackerListWidget::loadTrackers()
 
         item->setText(COL_RECEIVED, QString::number(data.numPeers));
 #if LIBTORRENT_VERSION_NUM >= 10000
-        item->setText(COL_SEEDS, QString::number(entry.nativeEntry().scrape_complete > 0 ? entry.nativeEntry().scrape_complete : 0));
-        item->setText(COL_PEERS, QString::number(entry.nativeEntry().scrape_incomplete > 0 ? entry.nativeEntry().scrape_incomplete : 0));
-        item->setText(COL_DOWNLOADED, QString::number(entry.nativeEntry().scrape_downloaded > 0 ? entry.nativeEntry().scrape_downloaded : 0));
+        item->setText(COL_SEEDS, entry.nativeEntry().scrape_complete > -1 ? QString::number(entry.nativeEntry().scrape_complete) : tr("Unknown"));
+        item->setText(COL_PEERS, entry.nativeEntry().scrape_incomplete > -1 ? QString::number(entry.nativeEntry().scrape_incomplete) : tr("Unknown"));
```

Comment:
Could "Not avaliable" (or "N/A") be better?

Reply:
Sure i like N/A. Should it be translatable?

Reply:
"N/A"

Reply:
>Should it be translatable?

Of course!